lib/param_parsing/toml.go

```diff
@@ -1,23 +1,24 @@
 package param_parsing
 
 import (
+	"path/filepath"
+
 	"github.com/spf13/viper"
 	"github.com/warrensbox/terraform-switcher/lib"
-	"path/filepath"
 )
 
 const tfSwitchTOMLFileName = ".tfswitch.toml"
 
 // getParamsTOML parses everything in the toml file, return required version and bin path
-func getParamsTOML(params Params) (Params, error) {
-	tomlPath := filepath.Join(params.ChDirPath, tfSwitchTOMLFileName)
-	if tomlFileExists(params) {
+func getParamsTOML(params Params, baseDir string) (Params, error) {
```

Comment:
I still think this can be better?

`getParamsTOML(params Params, baseDir string)`

for:

params.ChDirPath = lib.GetHomeDirectory() //set path to home
params, err = getParamsTOML(params, lib.GetHomeDirectory())

that way we only need `getParamsTOML(params)`?

Reply:
If we don't respect the ChDirPath that the user passes, wouldn't it it be odd to manipulate the variable to use it for something that it's not intended for?